draft-irtf-cfrg-opaque.md

```diff
@@ -371,20 +371,32 @@ OPAQUE relies on the following protocols and primitives:
 Note that we only need the base mode variant (as opposed to the verifiable mode
 variant) of the OPRF described in {{I-D.irtf-cfrg-voprf}}.
 
-- Cryptographic hash function:
-  - Hash(m): Compute the cryptographic hash of input message `m`. The type of the
-    hash is determined by the chosen OPRF group.
-  - Nh: The output size of the Hash function.
-
-- Authenticated Key Exchange (AKE, {{instantiations}}):
-  - Npk: The size of the public keys used for the key exchange protocol.
-  - Nsk: The size of the private keys used for the key exchange protocol.
+- Key Derivation Function (KDF):
+  - Extract(salt, ikm): Extract a pseudorandom key of fixed length `Nl` bytes from
+    input keying material `ikm` and an optional byte string `salt`.
+  - Expand(prk, info, L): Expand a pseudorandom key `prk` using optional string `info`
+    into `L` bytes of output keying material.
+  - Nl: The output size of the `Extract()` function in bytes.
+
+- Message Authentication Code (MAC):
+  - MAC(key, msg): Compute a message authentication code over input `msg` with key
+    `key`, producing a fixed-length output of `Nm` bytes.
```

Comment:
nit: you reference `Nm` here, but it is defined below -- I'd say just remove the reference